README.md

```diff
@@ -46,11 +46,10 @@ from etna.models import ProphetModel
 
 # Read the data
 df = pd.read_csv("example_dataset.csv")
```

Comment:
I think this path should be smth like `examples/data/example_dataset.csv`, shouldn't it?

Reply:
Sounds reasonable)